### ISSUE TITLE
request support for pinterest

### ISSUE THREAD

**Issue by teel-writ-ros** [delucis/astro-embed#172]
request support for the ability to embed pinterest posts like the following.

`https://www.pinterest.com/yiwang731135/hengrui-canteen/`

thanks

**Comment by delucis** [comment]
Thanks for the request. Noting that in general the aim for this library is zero or low-JS components for better web performance, so we probably wouldn’t implement it just as an `<iframe>` or using Pinterest’s default client bundle.

Instead, maybe it’s possible to use the Pinterest API to fetch data about a post (e.g. https://developers.pinterest.com/docs/api/v5/pins-get/) and display that content.

**Comment by teel-writ-ros** [comment]
> Thanks for the request. Noting that in general the aim for this library is zero or low-JS components for better web performance, so we probably wouldn’t implement it just as an `<iframe>` or using Pinterest’s default client bundle.
> 
> Instead, maybe it’s possible to use the Pinterest API to fetch data about a post (e.g. https://developers.pinterest.com/docs/api/v5/pins-get/) and display that content.

certainly, looking forward to an implementation other than <iframe>, which doesn't work for now btw.
